tests/test_scheduler.rs

```diff
@@ -660,3 +666,35 @@ fn assert_sequentiality(
     dir_entries_second_plan.sort();
     assert!(dir_entries_first_plan[0] < dir_entries_second_plan[0]);
 }
+
+async fn assert_robocorp_home(
+    robocorp_home_base: &Utf8Path,
+    #[cfg(windows)] headed_user_name: &str,
+) -> AnyhowResult<()> {
+    assert!(robocorp_home_base.is_dir());
+    assert_eq!(
+        directory_entries(robocorp_home_base, 1),
+        [
+            "current_user",
+            #[cfg(windows)]
+            &format!("user_{headed_user_name}"),
+        ]
+    );
+    #[cfg(windows)]
+    {
+        let permissions_robocorp_home_base = get_permissions(robocorp_home_base).await?;
+        assert_eq!(
```

Comment:
It  does not really feel this is testing something?

Reply:
It seems quite tricky to test localization-independent that only the administrator group has access. Any suggestions?

Reply:
From a quick look, it appears `/findsid` and `/q` are relevant parameters to `icacls`. Otherwise, we could consider switching to the `win32` API or removing the check altogether.

Reply:
/findsid is what I needed. /q does not seem to have any effect on my VM, see also the comment in the code